src/generator/templates/api/lib/rbuild.mako

```diff
@@ -91,7 +91,7 @@ impl${rb_params} ${ThisType} {
         arg_prefix = "/// * `" + p.name + "` - "
 %>\
     ${arg_prefix}${p.get('description', "No description provided.")
-        | remove_empty_lines, prefix_all_but_first_with(' ' * SPACES_PER_TAB + '///'  + ' ' * (len(arg_prefix) - len('///')))}
+        | fix_relative_links(documentationLink), remove_empty_lines, prefix_all_but_first_with(' ' * SPACES_PER_TAB + '///'  + ' ' * (len(arg_prefix) - len('///')))}
```

Comment:
There's no `rust_doc_sanitize` call here. I wonder if it should be here and if we want to unify it with `fix_relative_links`?